packages/extensions/openapi-to-cadl/src/emiters/emit-cadl-config.ts

```diff
@@ -3,8 +3,8 @@ import { formatFile } from "../utils/format";
 
 export async function emitCadlConfig(filePath: string): Promise<void> {
   const content = `
-  emitters:
-    "@azure-tools/typespec-autorest": true
+  emit:
+    - "@azure-tools/typespec-autorest": true
```

Comment:
This should be updated:
```suggestion
    - "@azure-tools/typespec-autorest"
```